tests/data/run_processing.json

```diff
@@ -1,20 +1,24 @@
 {
     "operation_platform": "abacus",
-    "project_fms_id": null,
+    "project_ext_id": null,
+    "project_ext_src": null,
     "project_name": "MOH-Q",
-    "run_fms_id": null,
+    "run_ext_id": null,
+    "run_ext_src": null,
     "run_name": "220420_A01433_0157_BHM3NHDSX2_MoHRun08-novaseq",
     "run_instrument": "novaseq",
     "run_date": "2022-04-20 00:00:00",
-    "patient": [
+    "specimen": [
```

Comment:
The modification in the run processing script has not been implemented yet.

Reply:
That's fine as we are not using it yet